examples/complete/main.tf

```diff
@@ -0,0 +1,10 @@
+module "datadog_monitors" {
+  source = "../../"
+
+  monitors_config_file = var.monitors_config_file
```

Comment:
Monitors should take a HCL datastructure. The user can call `yamldecode(readfile(...))`

Reply:
Using HCL enforces YAML datastructure.